[DOC] Add docs for SORTABLE (#2678)
RediSearch/RediSearch: docs/commands/ft.aggregate.md

  each property. `nargs` is the number of sorting parameters, including ASC and DESC. for example:
  `SORTBY 4 @foo ASC @bar DESC`.

>   Attributes needed for sortby should be stored as **SORTABLE**, where they are available with very low latency.

Reviewer: ```suggestion
  Attributes needed for **SORTBY** should be stored as **SORTABLE** in order to be available with very low latency.
```
